modules/security-audit-log-filtering.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * security/audit-log-view.adoc
+
+[id="security-audit-log-basic-filtering_{context}"]
+= Filtering audit logs
+
+You can configure the audit log policy to control the amount of information to log to API server audit logs. If you need to filter the logs further, you can use `jq` or another JSON parsing tool.
```

Comment:
_to control the amount of information to log to API server audit logs_
This seems a little awkward; maybe the repetition of _to_.

Do we want to use the wording in the audit log policy topic?
You can configure the audit log policy to _**define how to log requests that come to the OpenShift API server**._

Reply:
Maybe add a link to the audit log policy topic somewhere?

Reply:
Does jq control what information is being logged or does it control what is displayed from the logs? I would think the latter, as that is my assumption about log filtering.

Reply:
I wonder if you want to switch the first two sentences where the topic is about filtering and policies control what is logged.
 
```
You can filter logs using `jq`j or another JSON parsing tool. 

[NOTE]
====
You can configure the audit log policy to control the amount of information to log to API server audit logs.
=====
```
Something like that so the emphasis is on the filtering.

Reply:
Good suggestions, thanks! I'll take a crack at rewording some.